src/components/Calculator.js

```diff
@@ -1,38 +1,62 @@
 import React, { PureComponent } from 'react';
+import calculate from '../logics/calculate';
 
 export default class Calculator extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      total: null,
+      next: null,
+      operation: null,
+    };
+  }
+
+  handleClick = (e) => {
+    const value = calculate(this.state, e.currentTarget.innerText);
+    this.setState(() => ({
+      total: value.total,
+      next: value.next,
+      operation: value.operation,
+    }));
+  }
+
   render() {
+    const { total, next, operation } = this.state;
     return (
       <div id="subroot">
-        <input id="input" value={0} />
+        <div id="input">
+          {total}
+          {operation}
+          {next}
+        </div>
         <div className="row1">
-          <button id="ac" type="button">AC</button>
-          <button id="negative" type="button">+/-</button>
-          <button id="modulus" type="button">%</button>
-          <button id="divide" className="col4" type="button">/</button>
+          <button id="ac" type="button" onClick={this.handleClick}>AC</button>
+          <button id="negative" type="button" onClick={this.handleClick}>+/-</button>
+          <button id="modulus" type="button" onClick={this.handleClick}>%</button>
+          <button id="divide" className="col4" type="button" onClick={this.handleClick}>/</button>
```

Comment:
- [ ] The division process is not working in your calculator app, Please use the `÷` sign instead of `/` to make it work properly.

![chrome-capture-2023-0-11](https://user-images.githubusercontent.com/68796232/211923649-571ec823-26eb-4527-a222-108c1e1d212f.gif)

Reply:
Thanks for finding this out. I am going to fix this.